cmake/platform-unix.cmake

```diff
@@ -3,4 +3,10 @@ INCLUDE(util)
 
 MESSAGE(STATUS "Configuring UNIX specific things and stuff...")
 
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/finder")
+
+include (FindLTTngUST REQUIRED)
```

Comment:
This library is not a platform library and should be linked directly into the code target instead.